src/mgr/DaemonServer.cc

```diff
@@ -1594,6 +1718,11 @@ bool DaemonServer::_handle_command(
     vector<string> ids;
     cmd_getval(cmdctx->cmdmap, "ids", ids);
     set<int> osds;
+    int64_t max = 1;
+    cmd_getval(cmdctx->cmdmap, "max", max);
+    if (max < (int)osds.size()) {
```

Comment:
shouldn't this be the opposite?

Reply:
This is for the case where you do ``ceph osd ok-to-stop osd.1 osd.2 osd.3 --max 2``.  In that case, we adjust max up to 3 (so that it is effectively a no-op).

Alternatively, we could error out with EINVAL...